docs/Reference/Server.md

```diff
@@ -952,6 +952,7 @@ Note that the array contains the `fastify.server.address()` too.
 #### getDefaultRoute
 <a id="getDefaultRoute"></a>
 
+The `defaultRoute` handler handles requests that do not match any URL specified by your Fastify application. This defaults to the 404 handler, but can be overridden with [setDefaultRoute](#setdefaultroute).
```

Comment:
Please hard wrap lines to be consistent with the rest of the document.

Reply:
done